packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -156,12 +190,25 @@ function usePositions(
   calloutElement: HTMLDivElement | null,
   targetRef: React.RefObject<Element | MouseEvent | Point | null>,
   getBounds: () => IRectangle | undefined,
+  popupRef: React.RefObject<HTMLDivElement>,
 ) {
   const [positions, setPositions] = React.useState<ICalloutPositionedInfo>();
   const positionAttempts = React.useRef(0);
   const previousTarget = React.useRef<Target>();
   const async = useAsync();
-  const { hidden, target, finalHeight, calloutMaxHeight, onPositioned, directionalHint } = props;
+  const {
+    hidden,
+    target,
+    finalHeight,
+    calloutMaxHeight,
+    onPositioned,
+    directionalHint,
+    hideOverflow,
+    preferScrollResizePositioning,
+  } = props;
+
+  const popupStyles = popupRef?.current ? window.getComputedStyle(popupRef.current) : undefined;
```

Comment:
Sounds good to me! Is it possible that the styles would change while popupRef.current points to the same object?

Reply:
> Is it possible that the styles would change while popupRef.current points to the same object?

Yes, but the [returned object is live](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle#return_value) so it will be updated too 🙂

Reply:
Oh makes sense! Thank you for the info! :)